chrome/nav.js

```diff
@@ -408,6 +398,7 @@ function configureMinimap( sector ) {
 	// Create the div that will hold distance calculations
 	div = doc.createElement( 'div' );
 	div.style.paddingTop = '4px';
+	div.style.height = '48px';
```

Comment:
This is part of a fix to the problem of the #rightSideBar moving when you hover over the minimap in sectors with long names.